_notices/rsn0036.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 36 # should match notice number
+notice_pin: false # set to true to pin to notice page
+title: "Support for Pandas 2.0 in Release v24.02"
+notice_author: RAPIDS TPM
+notice_status: In Progress
+notice_status_color: Yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v24.02+"
+notice_created: 2023-12-06
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-12-06
+---
+
+## Overview
+
+Pandas 1.x series support will be removed in the upcoming release of v24.02 scheduled for Februry 8, 2024.  Starting effectively in release v24.04 scheduled for April 11, 2024 the minimum Pandas suport will be 2.0 for RAPIDS 

```

Comment:
```suggestion
Pandas 1.x series support will be deprecated in the upcoming release of `v24.02` scheduled for February 8, 2024, and removed in `v24.04`, scheduled for April 11, 2024. Effective `v24.04`, the minimum version of Pandas supported will be Pandas 2. 

```

Reply:
Do we want to decide on a specific minimum version of Pandas to target here? Might be nice to give a specific version rather than "Pandas 2". @shwina, thoughts?

Reply:
I can talk for @shwina here, we can support every version starting from `2.0.0`